ttkwidgets/itemscanvas.py

```diff
@@ -103,7 +105,25 @@ def left_motion(self, event):
         rectangle = self.items[item]
         self.config(cursor="exchange")
         self.canvas.itemconfigure(item, fill="blue")
-        x, y = self.canvas.canvasx(event.x), self.canvas.canvasy(event.y)
+        xc, yc = self.canvas.canvasx(event.x), self.canvas.canvasy(event.y)
+        dx, dy = xc - self.current_coords[0], yc - self.current_coords[1]
+        self.current_coords = xc, yc
+#        x = self._max_x if x > self._max_x else x
+#        y = self._max_y if y > self._max_y else y
+#        x = 0 if x < 0 else x
+#        y = 0 if y < 0 else y
+        self.canvas.move(item, dx, dy)
+        self.canvas.move(rectangle, dx, dy)
+#        x1, y1, x2, y2 = self.canvas.coords(rectangle)
```

Comment:
These comments contain calculations for the new coordinates of the rectangle. However, the coordinates are calculated by putting the rectangle where the text is. That makes these comments a bit redundant.

Reply:
I had just commented out the old code and forgot to remove it. It's done now.